NineChronicles.Headless/GraphTypes/States/AvatarStateType.cs

```diff
@@ -110,6 +110,16 @@ public AvatarStateType()
                 nameof(AvatarState.combinationSlotAddresses),
                 description: "Address list of combination slot.",
                 resolve: context => context.Source.AvatarState.combinationSlotAddresses);
+            Field<NonNullGraphType<ListGraphType<NonNullGraphType<CombinationSlotStateType>>>>(
+                "combinationSlotState",
```

Comment:
Because combination slots are plural, how about naming this field `combinationSlotStates` or `combinationSlots`? I love naming like `agent`, and `avatar` instead of `avatarStates`.